python/marvin/tools/cube.py

```diff
@@ -177,8 +178,7 @@ def _load_cube_from_file(self, data=None):
             assert isinstance(data, fits.HDUList), 'data is not an HDUList object'
         else:
             try:
-                with gunzip(self.filename) as gg:
-                    self.data = fits.open(gg.name)
+                self.data = fits.open(self.filename)
```

Comment:
`gunzip` is a context manager designed to temporarily unzip the files before opening them.  This speeds up file loading.  If gzip is not available on Windows, then a suitable alternative for Windows should be implemented.  It looks like that functionality has been removed altogether.

Reply:
This is a mistake, that @mstalbot had there for some reason he described as "debugging", but I asked him to remove it, and I thought he did.